src/notifications/use-on-push-notification-opened.ts

```diff
@@ -20,10 +20,12 @@ export function useOnPushNotificationOpened() {
         );
         return;
       }
+      if (!navigation.isFocused()) return; // avoid navigating away from e.g. login or permission screens
```

Comment:
Correct. For example if a permission screen is showing, but then the user exits the app, and reopens it through a notification, then this check ensures that it will not navigate away from the permission screen.

Reply:
This also means that the linking will not work if screens like location search or fare contract details are open. Not saying it necessarily is wrong, just mentioning it so are aware. We have around 35 different screens in the app on root level which will make the notification link not work.

Reply:
Yeah. Could be that notification links should be allowed from some screens and not from others. If so we need something different than this isFocused check. Also possible to remove this check, assuming that the problematic navigates are very rare and acceptable.